projects/com.oracle.truffle.llvm.parser.bc.impl/src/com/oracle/truffle/llvm/parser/bc/impl/LLVMBitcodeVisitor.java

```diff
@@ -353,4 +357,8 @@ public void visit(FunctionDefinition method) {
     @Override
     public void visit(Type type) {
     }
+
+    public LLVMTypeHelper getTypeHelper() {
```

Comment:
Could we expose this method in `LLVMParserRuntime`? Then we could leave the node factories stateless.